.github/workflows/test.yml

```
@@ -99,6 +99,43 @@ jobs:
       - name: Run acceptance tests
         run: make test
 
+  acceptance-tests-cluster:
```

How about we go with shorter job names?
```suggestion
  acceptance-cluster:
```
And also `s/acceptance-tests/acceptance-standalone/`?